### PR TITLE
Upgrade minimist to avoid security vulnerability

### PR DESCRIPTION
**Pull request by mondeja**
See [this](https://github.com/substack/minimist#security) for more info.

### CODE REVIEW

**Comment by A----** [comment]
Additional resources: https://www.npmjs.com/advisories/1179

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
   "dependencies": {
     "xmldom": "~0.1.19",
     "concat-stream": "~1.5.1",
-    "minimist": "1.2.0"
+    "minimist": "1.2.3"
```
Comment (A----) [inline package.json:42]:
```suggestion
    "minimist": "~1.2.3"
```

Should work & allow future minors

**Comment by A----** [comment]
Also, fixes #163

**Comment by andrewharvey** [comment]
you might want to checkout https://github.com/tmcw/togeojson which is a maintained fork of togeojson.

**Comment by mondeja** [comment]
> you might want to checkout https://github.com/tmcw/togeojson which is a maintained fork of togeojson.

Thanks!

**Pull request closed without merging**